src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Api.FakeServers/CommitmentsOuterApiBuilder.cs

```diff
@@ -48,18 +48,6 @@ internal CommitmentsOuterApiBuilder WithCourseDeliveryModels()
                         DeliveryModels = new[] { "Regular", "PortableFlexiJob" },
                     }));
 
-            _server
```

Comment:
Do we not need one course that has only Regular so we can test the journey that skips the delivery model page?